scripts/packaging/createupmpackages.ps1

```diff
@@ -22,7 +22,7 @@ param(
     [string]$Version,
     [ValidatePattern("^\d+?[\.\d+]*$")]
     [string]$BuildNumber,
-    [bool]$IsOfficial = $False
+    [bool]$UseBuildNumber = $True

```

Comment:
This might also work with [switch] to simplify having to pass a 0 or 1:

https://stackoverflow.com/questions/5079413/how-to-pass-boolean-values-to-a-powershell-script-from-a-command-prompt/5079574#5079574

Reply:
i tested, passing 0 and 1 do work. i just like being explicit inside the script :)

Reply:
I get that they do work - my statement here is that it's really unexpected given regular command line args.

i.e. boolean args are usually --IsOfficial which means it's true, the absence == false.

That's why switch could be interesting to use.

This goes along with another change later which uses a string "true" instead of 0 or 1 and is inconsistent.

Let's use switch unless that doesn't work.

Reply:
let me give that a shot

Reply:
using a switch arg now for whether or not to include the build number in the package version.

the remaining "true" / "false" checks are due to the reuse of the existing publishingEnabled pipeline parameter (string)